.env.dev

```
@@ -0,0 +1,13 @@
+# NESTJS CONFIG
+NESTJS_PORT=9000
+
+# NATS.IO CONFIG
+NATS_CLIENTS_URL=nats://identityevents-dev-nats.energyweb.org:4222
+
+REQUEST_NEW_ORG_ROLE=mytest.roles.testy.iam.ewc
+VOLTA_CHAIN_ID=73799
+CACHE_SERVER_URL=https://identitycache-dev.energyweb.org/
+RPC_URL=https://volta-rpc.com
```

better to have it be actual RPC url
```suggestion
RPC_URL=https://volta-rpc.energyweb.org
```